notifications.bs

```diff
@@ -83,6 +83,14 @@ associated <dfn>icon URL</dfn>, <dfn>icon resource</dfn>, <dfn>sound URL</dfn>,
 <p class=note>Developers are encouraged to not convey information through an icon, sound,
 or vibration pattern that is not otherwise accessible to the end user.
 
+<p>A <a lt="concept notification">notification</a> has an associated list of
+<dfn>actions</dfn> (which may be empty). Platforms may render these as buttons.
```

Comment:
I prefer "action list" as single `<dfn>`. And then "consisting of zero or more actions".